ogcore/parameters.py

```diff
@@ -3,6 +3,7 @@
 import scipy.interpolate as si
 import pkg_resources
 import paramtools
+from itertools import chain
```

Comment:
@jdebacker. I don't see this `chain` module used anywhere in `parameters.py`. If that is the case, we should remove this.